.travis.yml

```diff
@@ -1,30 +1,34 @@
 language: android
-dist: trusty
 
 android:
   components:
-    - tools                             # Tools
-    - platform-tools                    # Platform tools
-    - build-tools-26.0.1                # Build tools version
-    - android-26                        # Target SDK version
-    - extra-android-m2repository        # Support repo
-    - sys-img-x86-android-18            # Emulator
+    # Update tools and then platform-tools explicitly so lint gets an updated database. Can be removed once 3.0 is out.
+    - tools
+    - platform-tools
 
 jdk:
   - oraclejdk8
 
-# Disabled, see https://github.com/uber/AutoDispose/issues/81
-#before_script:
-#  # Create and start an emulator for instrumentation tests.
-#  - echo no | avdmanager create avd -f -n test -k "system-images;android-18;google_apis;x86"
-#  - emulator -avd test -no-audio -no-window &
-#  - android-wait-for-emulator
-#  - adb shell input keyevent 82
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "8933bad161af4178b1185d1a37fbf41ea5269c55" > "$ANDROID_HOME/licenses/android-sdk-license"
```

Comment:
you should move this sha to the travis ci env variables in the job setup

Reply:
bc11c4c